src/avalanchemq/http/controller/prometheus.cr

```diff
@@ -172,7 +172,7 @@ module AvalancheMQ
           unacked += d[:messages_unacknowledged]
           connections += vhost.connections.size
           vhost.connections.each do |conn|
```

Comment:
I think the problem is this, iterating all quarter million connections. `channels.size` is already a "property" today so i don't think this makes much of a difference. What we want is a `channel_count` property in `Vhost`.

Reply:
One option is to not count channels if there are to many connections. But I can look at how it can be stored on vhost level.

Reply:
yes, that might be a quickfix!